README.md

```diff
@@ -10,3 +10,15 @@ file. Simply add a ```build``` directory to the project directory and navigate t
 then flash to your Core/Photon using ```particle flash DEVICEID firmware.bin```
 
 Note that if you do this, you'll need to have CMake. If you're using CLion, CMake should be bundled within the IDE directory somewhere.
+
+
+### Sensor_*
+Ported AVR code for interacting with DS18B20 sensors.
+
+Supports multiple sensors on one pin.
+
+Standard hardware configuration, 4.7k ohm resistor from data to VCC line.
+
+Note that this is C code, not CPP, so functions etc are not enclosed in their own name space.
+The functions are named crc8 for the CRC calculations, ow_ for onewire functions, and DS18X20_ for temperature specific functions.
```

Comment:
It looks like you copied most of this new stuff directly from the README for the particle-ds18x20 library. I don't think it's really necessary here - if you need to know how to use the library correctly, you're going to look at the library itself.